OpenRA.Mods.Common/SpriteLoaders/ShpTSLoader.cs

```diff
@@ -118,7 +118,9 @@ bool IsShpTS(Stream s)
 			while (w == 0 && h == 0 && f++ < imageCount);
 
 			s.Position = start;
-			return type < 4;
+
+			// Type 18 is a special case for RA2 low bridges.
```

Comment:
Are you sure that the test method is being called with the correct data (and the correct stream position)?  30,19 doesn't look like a reasonable frame size for a 3x1 diagonal bridge tile, so I suspect you may be parsing random bytes.